fe/src/project/AspNetCore/areas/DownloadCenter/components/DownloadCenterCatalog/DownloadCenterCatalog.scss

```diff
@@ -0,0 +1,17 @@
+.dc-catalog {
+    &__container {
+        margin-bottom: 3rem;
+
+        &:last-child {
+            margin-bottom: 0;
+        }
+    }
+
+    &__list-item {
+        background: #f7f7f7;
```

Comment:
**suggestion:** let's put the '#f7f7f7' to variables and reference the name here